recipe/meta.yaml

```diff
@@ -220,28 +225,31 @@ outputs:
         - $RPATH/ld64.so.2  # [ppc64le]
     requirements:
       build:
-        - {{ compiler('m2w64_c') }}    # [win]
-        - {{ compiler('m2w64_cxx') }}  # [win]
-        - {{ compiler('c') }}
-        - {{ compiler('cxx') }}
-        - {{ stdlib('c') }}
-        - llvm-openmp                  # [osx]
-        - libgomp                      # [linux]
+        - {{ compiler('m2w64_c') }}        # [win]
+        - {{ compiler('m2w64_cxx') }}      # [win]
+        - {{ compiler('m2w64_fortran') }}  # [win]
+        - {{ stdlib('m2w64_c') }}          # [win]
+        - {{ compiler('c') }}              # [unix]
+        - {{ compiler('cxx') }}            # [unix]
+        - {{ stdlib('c') }}                # [unix]
```

Comment:
Maybe we can order these to match the top-level list of compilers. Think this could help when making sure they have similar things

Reply:
Agreed, this recipe is hard enough to follow as is so any readability improvements will help.